website/profile/views.py

```diff
@@ -284,6 +284,10 @@ def user_account_password(auth, **kwargs):
     new_password = request.form.get('new_password', None)
     confirm_password = request.form.get('confirm_password', None)
 
+    if not throttle_period_expired(user.change_password_last_attempt, settings.CHANGE_PASSWORD_THROTTLE):
+        raise HTTPError(httplib.BAD_REQUEST,
```

Comment:
Minor thing, when you're throttling requests the proper HTTP status code is [429](https://httpstatuses.com/429).

Reply:
Ended up pushing a status message anyway, instead of showing a new page with Bad Request.  When the old password is incorrect, it also pushes a status message, so thought this would be more consistent with other error behavior on this form.

Reply:
I'm not sure there should be any user facing aspect to this. My understanding is that it's just to stop people from resending the request multiple times via the browser or externally.  Using push notications when doing this can create misleading messages on refresh like the picture below, but no typical user would get this notification in the first place, so I'll pass.
<img width="1439" alt="screen shot 2018-03-13 at 3 37 00 pm" src="https://user-images.githubusercontent.com/9688518/37365826-03038c6e-26d5-11e8-9357-f65607595698.png">